packages/stacks-classic/lib/test/assertions.ts

```diff
@@ -32,5 +32,45 @@ const WCAGNonTextContrast: AdditionalAssertion = {
     },
 };
 
+const WCAGBoxShadowContrast: AdditionalAssertion = {
```

Comment:
I am working on the Navigation component which have a similar way to highlight selected state but in there we won't be able to use the box shadow directly to the element. Instead we will likely need a pseudo element. All of this to say that it might be sensible to remove the check all together from this PR and have this code you created in a branch and linked to a new ticket with context. The assertion we have at the moment is too specific and should probably be generalized. If we cannot do it in a sensible way maybe even accept that it is not worth testing. 

In a nutshell my recommendation would be to extract this in a new branch and create a related ticket for the backlog. Thanks @ttaylor-stack for experimenting with this. ❤️